Libplanet.Net/Consensus/ConsensusContext.cs

```diff
@@ -52,14 +59,29 @@ public class ConsensusContext<T> : IDisposable
         /// <param name="lastCommitClearThreshold">A maximum size of cached
         /// <see cref="BlockCommit"/>. See <see cref="LastCommitClearThreshold"/>.
         /// The value must bigger than <c>0</c>.</param>
+        /// <param name="timeoutProposeSecond">A timeout of Propose <see cref="Step"/>.</param>
+        /// <param name="timeoutPreVoteSecond">A timeout of PreVote <see cref="Step"/>.</param>
+        /// <param name="timeoutPreCommitSecond">A timeout of PreCommit <see cref="Step"/>.</param>
+        /// <param name="timeoutProposeMultiplier">A multiplier value of Propose <see cref="Step"/>
+        /// timeout for when the <see cref="Round"/> goes up.</param>
+        /// <param name="timeoutPreVoteMultiplier">A multiplier value of PreVote <see cref="Step"/>
+        /// timeout for when the <see cref="Round"/> goes up.</param>
+        /// <param name="timeoutPreCommitMultiplier">A multiplier value of PreCommit
+        /// <see cref="Step"/> timeout for when the <see cref="Round"/> goes up.</param>
         public ConsensusContext(
             DelegateBroadcastMessage broadcastMessage,
             BlockChain<T> blockChain,
             long height,
             PrivateKey privateKey,
             TimeSpan newHeightDelay,
             Func<long, IEnumerable<PublicKey>> getValidators,
-            long lastCommitClearThreshold)
+            long lastCommitClearThreshold,
+            int timeoutProposeSecond = 5,
+            int timeoutPreVoteSecond = 5,
+            int timeoutPreCommitSecond = 5,
+            int timeoutProposeMultiplier = 1,
+            int timeoutPreVoteMultiplier = 1,
+            int timeoutPreCommitMultiplier = 1)
```

Comment:
I think these should be grouped together as a `class` for following reasons:

- Moving away from `const` allows nonsensical values to be passed such as negative values.
- Adding six more parameters to what is already seemingly too cluttered.

Also this seems like a nice sensible unit to make a small separate PR. 🙂